dlt/cli/config_toml_writer.py

```diff
@@ -26,25 +33,16 @@ def generate_typed_example(name: str, hint: AnyType) -> Any:
     inner_hint = extract_inner_hint(hint)
     try:
         sc_type = py_type_to_sc_type(inner_hint)
-        if sc_type == "text":
-            return "<configure me>"
+        if sc_type in TYPE_EXAMPLES.keys():
+            return TYPE_EXAMPLES[sc_type]
         if sc_type == "bigint":
             return 0
         if sc_type == "double":
             return 1.0
         if sc_type == "bool":
             return True
         if sc_type == "json":
-            if is_subclass(inner_hint, C_Sequence):
-                return ["a", "b", "c"]
-            else:
-                table = tomlkit.table(False)
-                table["key"] = "value"
-                return table
-        if sc_type == "timestamp":
-            return pendulum.now().to_iso8601_string()
-        if sc_type == "date":
-            return pendulum.now().date().to_date_string()
+            return "json"
```

Comment:
does this produce a useful result? if you don't want anything to be written for json (which I guess makes sense) then I think you need to skip this value somehow.

Reply:
first i was raising an not implemented error here, but I thought its cleaner to produce something valid here and deal with it in the place that this function is used.
 (its only one place)
![image](https://github.com/user-attachments/assets/42110098-9913-4767-861e-8e712a0cdaa6)